hello/hello_signal.py

```diff
@@ -26,7 +26,9 @@ async def run(self) -> List[str]:
             while not self._pending_greetings.empty():
                 greetings.append(f"Hello, {self._pending_greetings.get_nowait()}")
 
-        return greetings
+            # Exit it complete
```

Comment:
```suggestion
            # Exit if complete
```

(made a mistake in my diff before, but not a big deal)